multimedia/tvheadend/Makefile

```diff
@@ -33,8 +33,10 @@ define Package/tvheadend
 	+librt \
 	+libffi \
 	+libopenssl \
+	+liburiparser \
```

Comment:
This would add an unconditional dependency on liburiparser.

Reply:
Thanks for your reminder, I have made the changes.